seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/catalog/psql/PostgresCatalog.java

```diff
@@ -186,8 +188,44 @@ protected Column buildColumn(ResultSet resultSet) throws SQLException {
     }
 
     @Override
-    protected String getCreateTableSql(TablePath tablePath, CatalogTable table) {
-        return new PostgresCreateTableSqlBuilder(table).build(tablePath);
+    protected void createTableInternal(TablePath tablePath, CatalogTable table)
+            throws CatalogException {
+        PostgresCreateTableSqlBuilder postgresCreateTableSqlBuilder =
+                new PostgresCreateTableSqlBuilder(table);
+        String createTableSql = postgresCreateTableSqlBuilder.build(tablePath);
+        String dbUrl = getUrlFromDatabaseName(tablePath.getDatabaseName());
+        Connection conn = getConnection(dbUrl);
```

Comment:
`executeInternal` method can be used here.

Reply:
Thanks for your reminder, update the code, please review again.